code/modules/admin/verbs/deadsay.dm

```diff
@@ -24,7 +24,7 @@
 	if (!msg)
 		return
 
-	var/rendered = "<span class='game deadsay'><span class='prefix'>DEAD:</span> <span class='name'>ADMIN([src.holder.fakekey ? pick("BADMIN", "hornigranny", "TLF", "scaredforshadows", "KSI", "Silnazi", "HerpEs", "BJ69", "SpoofedEdd", "Uhangay", "Wario90900", "Regarity", "MissPhareon", "LastFish", "unMportant", "Deurpyn", "Fatbeaver") : src.key])</span> says, <span class='message'>\"[msg]\"</span></span>"
```

Comment:
surely we can just axe this one entirely, or put in our own joke admin names... former admin list perhaps?

Reply:
That's the stealthmin nonsense so you can just replace it with "stealthed" or something... or put the name of the notemaster as the one true stealthmaster

Reply:
darkhrose...

Reply:
that is a list of very old admins

Reply:
Assumed they were old goon or baymins, wew.

AFAIK not even one of these guys was still around when I started in late 2014

Reply:
Some of them are TG admins